packages/larva-css/src/01-generic/typography.common.inline.scss

```diff
@@ -1,11 +1,15 @@
 @import 'setup';
 
+.wf-active {
+	--font-family-primary: var( --font-family-primary-fancy );
+	--font-family-secondary: var( --font-family-secondary-fancy );
+}
+
 body {
 
-	// @todo: this is outside the system and it should use the font-family mixin.
-	// That mixin needs to support `&.wf-active`, right now it only supports `.wf-active &`.
 	// stylelint-disable property-blacklist
-	font-family: Helvetica, sans-serif;
+	font-family: map-get( $TOKENS-MAP, font-family-body );
```

Comment:
@sarahsmo Looking at this with fresh eyes/brain, I think we would want `font-family-basic` to be the default instead of body – the body font family is intended for copy and will usually be a serif. Can you update this to `font-family-basic` please?